app/assets/javascripts/application/notifications.js

```diff
@@ -0,0 +1,8 @@
+Utils.fireWhenReady(['notifications#index', 'notifications#notifications'], function(e) {
+  $('.approve-sign-slide').on('click', function(event) {
+    var message = $(this).parent().siblings('.input-field').children('input').val();
+    $(this).attr('data-message', message);
+    $.post($(this).attr('href') + "?message=" + message );
+    return false;
+  });
+});
```

Comment:
Duplicated? I think the js in the /application folder is only included on the sign_controller#play action.